tools/tests/com.liferay.ide.gradle.core.tests/src/com/liferay/ide/gradle/core/tests/GradleParseTests.java

```diff
@@ -0,0 +1,65 @@
+package com.liferay.ide.gradle.core.tests;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+import org.codehaus.groovy.control.MultipleCompilationErrorsException;
+import org.junit.Test;
+
+import groovyjarjarantlr.RecognitionException;
+import groovyjarjarantlr.TokenStreamException;
+
+/**
+ *
+ * @author Lovett Li
+ *
+ */
+public class GradleParseTests
+{
+
+    final static File file = new File( "projects/testParse.gradle" );
+
+    @Test
+    public void addDependencies() throws IOException, RecognitionException, TokenStreamException
+    {
+        GradleScriptASTParser gradleScriptASTParser = new GradleScriptASTParser( new FileInputStream( file ) );
+
+        GradleScriptASTVisitor visitor = gradleScriptASTParser.walkScript();
+
+        int dependenceLineNum = visitor.getDependenceLineNum();
```

Comment:
We need to assert the dependencyLineNum that is 27, not another one of the other dependencies blocks